src/lib/common/limits.h

```diff
@@ -65,6 +65,8 @@
 #define IP_LENGTH_MAX           15    // Based on xxx.xxx.xxx.xxx
 #define STRING_SIZE_FOR_INT     16    // Room enough for an integer
 #define STRING_SIZE_FOR_DOUBLE  64    // Room enough for a double
+#define CORRELATOR_ID_SIZE      36
```

Comment:
I thought about that, but decided that CORRELATOR_ID_SIZE explains pretty well what it is all about.  STRING_SIZE_FOR_DOUBLE also. That comment (Room enough for a double) is pretty useless ... :-)

However, for symmetry, I'll add a comment for it